styles.css

```diff
@@ -0,0 +1,150 @@
+             /* ------------header------------ */
+             .logo-box{
+                display: flex;
+                width: 187px;
+                height: 58px;
+                align-items: center;
+                padding: 13px 57px 13px 0px;
+            }
+
+
+            .right-nav li a{
+                text-decoration: none;
+                color: #ffffff;
+            }
+
+            .right-nav{
+                display: flex;
+                font-size: 14px;
+                font-style: normal;
+                font-weight: 600;
+                line-height: 24px;
+                align-items: center;
+                gap: 21px;
+                list-style: none;
+                padding: 0%;
+            }
+
+            .right-nav-icons{
+                display: flex;
+                align-items: center;
+                gap: 18px;
+                list-style: none;
+                padding: 0%;
+            }
+
+            .navbar{
+                display: flex;
+                flex-direction: row ;
+                align-items: end;
+                padding: 13px 0px 13px 0px;
+                gap: 39px;
+            }
+
+            .header{
+                display: flex;
+                color: #fff;
+                flex-direction: row;
+                align-items: center;
+                padding: 23px 164px 23px 136px;
+                margin: auto;
+                width: 70%;
+                justify-content: space-between;
+            }
+
+            
+             /* ------------container------------ */
+            h1{
+                max-width: 75%;
+                text-align: center;
+                font-size: 4em;
+                font-weight: 800;
+                line-height: 80px;
+                letter-spacing: 0.2px;
+                color: #ffffff;
+                margin: 0px;
+            }
+
+            h4{
+                max-width: 71%;
+                text-align: center;
+                font-size: 1.3em;
+                font-weight: 500;
+                line-height: 30px;
+                letter-spacing: 0.2px;
+                color: #fff;
+                margin: 0px;
+            }
+
+            .button-get{
```

Comment:
YOu have 2 similar buttons. Only colors are different.
I highly recommend you give them 2 classes to each. Ex: "bottom button-get" nad "button button-learn" 
Give all common styles to "button" class and use custom class for custom styles.

This advice works for all cases where you have similar elements with small amount of differences